auth-web/src/views/auth/AcceptInviteView.vue

```diff
@@ -61,7 +63,9 @@ export default class AcceptInviteView extends Mixins(NextPageMixin) {
       } else {
         const invitation = await this.acceptInvitation(this.token)
         ConfigHelper.addToSession(SessionStorageKeys.CurrentAccount, JSON.stringify({ id: invitation.membership[0].org.id }))
+        await this.syncMembership(invitation.membership[0].org.id.toString())
```

Comment:
Should we use some optional chaining here to handle empty memberships?

Reply:
Done